testing/generate-mock-info

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+import unittest.mock
+
+import mock
+
+
+def main() -> int:
+    mock_all = mock.__all__  # type: ignore
+    unittest_mock_all = unittest.mock.__all__  # type: ignore
+    mock_version = mock.__version__  # type: ignore
+
+    renames_s = '\n'.join(
+        f"    'mock=unittest.mock:{attr}',"
+        for attr in sorted(set(mock_all).intersection(unittest_mock_all))
+    )
+
+    print(
+        # f'# GENERATED VIA {os.path.basename(sys.argv[0])}\n'
```

Comment:
is this supposed to be dead code?

Reply:
Oops, commented this out accidentally while debugging the output..